packages/did-provider-oyd/src/oyd-did-provider.ts

```diff
@@ -102,7 +102,7 @@ export class OydDIDProvider extends AbstractIdentifierProvider {
   private async holdKeys(args: OydDidHoldKeysArgs, context: IContext): Promise<IKey> {
     if (args.options.privateKeyHex) {
       return context.agent.keyManagerImport({
-        kms: args.kms || this.defaultKms,
+        kms: args.kms || this.defaultKms || '',
```

Comment:
Rather have a ts-ignore here. Our KMS handles it correctly when no KMS is provided. Using an empty string will result in errors